src/rotation/utils/holiday.ts

```diff
@@ -86,10 +95,48 @@ export async function getHolidayArray(): Promise<HolidayInfo[]> {
           day: +locdateString.slice(6, 8),
           info: item.dateName || 'null',
         };
+
         holidayArray.push(holidayInfo);
+        return holidayArray;
       }
 
-      return holidayArray;
+      if (data.response.body.totalCount > 2) {
```

Comment:
```suggestion
      if (data.response.body.totalCount >= 2) {
```
2개 초과가 아니라 2개 이상 아닐까요?!

Reply:
감사합니다... 큰일날뻔했네요...